Sillycore.EntityFramework/Sillycore.EntityFramework.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
 	<PackageId>Sillycore.EntityFramework</PackageId>
-    <PackageVersion>5.6.1</PackageVersion>
+    <PackageVersion>6.0.0</PackageVersion>
```

Comment:
There are no breaking chnages so no need to update major verion.

Reply:
We are going to rollback this, since we change the code to be backward compatible.